apps/devhub/views.py

```diff
@@ -1777,8 +1777,23 @@ def api_key(request):
         new_credentials = APIKey.new_jwt_credentials(request.user)
         log.info('new JWT key created: {}'.format(new_credentials))
 
+        send_key_change_email(request.user.email, new_credentials.key)
+
         return redirect(reverse('devhub.api_key'))
 
     return render(request, 'devhub/api/key.html',
                   {'title': _('Manage API Keys'),
                    'credentials': credentials})
+
+
+def send_key_change_email(to_email, key):
+    subject = '[Mozilla Addons] New API key created'
```

Comment:
I don't think you need to include brackets. People can make filters based on the sender address.